.vscodeignore

```diff
@@ -11,6 +11,7 @@ extension/tsconfig.json
 extension/vsc-extension-quickstart.md
 extension/node_modules/**
 test_case/**
+test-resources/**
 vendor/**
```

Comment:
I see we added the package `vscode-extension-tester`, it is best to filter out from the vscode extension pacakge, to minimize the technical debt.

Reply:
I didn't find it in the vsix that was made by this tester tool during the e2e test:
![image](https://user-images.githubusercontent.com/57521938/112925901-7cb41380-90c7-11eb-92d6-4b1ff9050974.png)

Reply:
just checked the vsix made by pack.py too, though it's bigger, it doesn't include this new package either.